app/controllers/settings/users/index.js

```diff
@@ -145,4 +128,12 @@ export default class UsersSettingsController extends Controller {
     }
     await Promise.all(this.membershipsBeingBlocked.map((membership) => membership.save()));
   }
+
+  @action
+  filterMemberships(memberships) {
```

Comment:
Can you add a comment for our future selves to clarify why we do the additional filtering in the frontend